src/dynamics/revolute_joint.rs

```diff
@@ -167,7 +167,7 @@ impl RevoluteJoint {
     /// The angle along the free degree of freedom of this revolute joint in `[-π, π]`.
     ///
     /// # Parameters
-    /// - `bodies` : the rigid body set from [`super::super::RapierContext`]
+    /// - `bodies` : the rigid body set from [`RapierContext`]
```

Comment:
Unrelated but this stemmed from an unconflicted PR after doc ci ; could've been avoided with a merge with master beforehand but I'll consider it minor enough.